client/my-sites/stats/hooks/use-stats-navigation-history.ts

```diff
@@ -39,6 +40,16 @@ const SUPPORTED_QUERY_PARAMS: string[] = [
 	'chartStart',
 	'chartEnd',
 	'shortcut',
+	'jp_s',
```

Comment:
The fixed list is here. It'd be even better to have fixed list from Jetpack side as well 😄